packages/react-dom/src/events/DOMPluginEventSystem.js

```diff
@@ -216,6 +219,7 @@ export const nonDelegatedEvents: Set<DOMEventName> = new Set([
   // and can occur on other elements too. Rather than duplicate that event,
   // we just take it from the media events array.
   ...mediaEventTypes,
```

Comment:
Note that adding `resize` here would disable delegation for it. Are there _any_ existing cases where we relied on delegation for it? I suppose not because we used to warn, so it was definitely unused.

Reply:
As far as I can tell it was unused. Perhaps there could be a future conflict if a Synthetic Event was created for the [Window:resize event](https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event)?

It seems safe to me for now, but let me know if there are any further tests you’d like me to run.